UI/src/model/selectedPointAsset.js

```diff
@@ -201,5 +243,22 @@
       });
       eventbus.trigger(assetName + ':changed');
     }
+
+    function getWasOldAsset() {
```

Comment:
at the point of the execution where this variable is used the asset is no longer an old traffic light, it has been converted to a new one but it still needs to keep some kind of trace saying is not a newly created asset but a converted one. Naming it "isOldAsset" is not an accurate name. And it might be confused with the variable "isOldTrafficLight" implemented in TrafficLightForm. I can change the name to "isConvertedAsset" this might be suggestive